plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -191,6 +193,19 @@ public void testSkipUnknownTypes()
         assertQueryReturnsEmptyResult("SHOW COLUMNS FROM test.tmp_guess_schema2");
     }
 
+    @Test
+    public void testDBRefSupport()
+    {
+        Document document = Document.parse("{\"_id\":ObjectId(\"5126bbf64aed4daf9e2ab771\"),\"col1\":\"foo\"}");
+
+        ObjectId objectId = new ObjectId("5126bc054aed4daf9e2ab772");
+        DBRef dbRef = new DBRef("test", "creators", objectId);
+        document.append("creator", dbRef);
+
+        client.getDatabase("test").getCollection("test_dbref").insertOne(document);
+        assertQuery("SELECT count(*) FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT 1");
```

Comment:
Please verify actual values instead of aggregated count.

Reply:
@ebyhr So i as doing the following:
        assertQuery("SELECT creator.collectionName FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT 'creators'"); == > WORKS
        assertQuery("SELECT creator.databaseName FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT 'test'"); ==> WORKS
        assertQuery("SELECT creator.id FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT '51 26 bc 05 4a ed 4d af 9e 2a b7 72'"); ==> **java.lang.AssertionError: unhandled type: ObjectId**

Should  I add the type to H2QueryRunner ? Or do you suggest i do it a different way ?

Reply:
was*

Reply:
Let's cast to varchar as existing method. Also, you don't need to separate into 3 `assertQuery`.

Reply:
added both count as well as varchar test query.

Reply:
The current test is not helpful when we change L207 as below subsequent comment. Let's verify the type instead. 
```java
        assertQuery(
                "SELECT typeof(creator) FROM test.test_dbref",
                "SELECT 'row(databaseName varchar, collectionName varchar, id ObjectId)'");
```

Reply:
Added